mediator/med.F90

```diff
@@ -345,6 +354,64 @@ subroutine SetServices(gcomp, rc)
          specPhaseLabel="med_fraction_set", specRoutine=med_fraction_set, rc=rc)
     if (ChkErr(rc,__LINE__,u_FILE_u)) return
 
+    !------------------
+    ! phase routines for budget diagnostics
+    !------------------
+
+    call NUOPC_CompSetEntryPoint(gcomp, ESMF_METHOD_RUN, &
+         phaseLabelList=(/"med_phases_diag_atm"/), userRoutine=mediator_routine_Run, rc=rc)
+    call NUOPC_CompSpecialize(gcomp, specLabel=mediator_label_Advance, &
+         specPhaselabel="med_phases_diag_atm", specRoutine=med_phases_diag_atm, rc=rc)
```

Comment:
It seems the med_phases_diag_atm will always run?

Reply:
It will only run if it is included in the run sequence.